README.md

```diff
@@ -41,7 +41,7 @@ NOTE: Be aware that the configuration in debian/rules expects tabs instead of sp
 
 Once the package is built, you have a virtualenv contained in a Debian
 package and upon installation it gets placed, by default, under
-`/usr/share/python/<packagename>`.
+`/opt/venvs/<packagename>`.
```

Comment:
The location is configurable so you can change it for yourself.

By the very nature of this tooling it violated the Debain packaging principles as its a "bring the whole environment" form of packaging. If you look at every other package that comes in this form, be it the Puppet AIO packaging, GitLab Omnibus and a bunch more, they all live in `/opt`.